<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>641</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>INFORMACIÓN DEL CLIENTE:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Código:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombres:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTelefono">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>90</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtApellido">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellidos:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnguardar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>GUARDAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnlimpiar">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>LIMPIAR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCodigo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNombre">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Telefono:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trabajo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTrabajo">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Direccion:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDireccion">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>140</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminar">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ELIMINAR</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DETALLE CLIENTE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>641</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>LISTADO DE CLIENTES:</string>
    </property>
    <widget class="QTableWidget" name="tblClientes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>611</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Cod.Cliente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellidos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Trabajo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefono</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Direccion</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
